docs/hosting/update.md

```diff
@@ -17,8 +17,9 @@ patches when they become available.
 2. **Update the farmOS codebase.** This will depend on how you have deployed
    farmOS. See [Updating via Docker](#updating-via-docker) or
    [Updating via packaged releases](#updating-via-packaged-releases) below for
-   specific instructions.
-3. **Run automated updates.** Navigate to `https://[hostname]/update.php` in
+   specific instructions. If you are building a custom farmOS codebase with
+   Composer see [/hosting/composer#updating-farmos]().
```

Comment:
Is this link broken?

Reply:
Yes! Good catch. :-)

Fixed here: https://github.com/farmOS/farmOS/pull/648/commits/5fa87ef326305bb8dc2d5698e204a5b140ba1e1b